### ISSUE TITLE
Airdog X3,X5 comparibility

### ISSUE THREAD

**Issue by MacDyson909** [Caldis/homebridge-airdog-air-purifier-x7sm#1]
Hello,
In first, thanks a lot for this plug-in, i'm was await it since 2 years ! :-)
I'm using a X3 model with youur plug-in, and all work, but not at 100%...
Maybe something you can modify  ? 
1 - X3 don't have VOC sensor... that's make a trouble for homekit... 
2 - X3 have only 4 speeds, 20 / 40 / 60 & 80 % of velocity.... that's make a trouble for homekit too... 
3 - The PM Value is not refreshed in homekit, only a reload of Home, provide the actual value... Automation can not work without ... Could you add a 5sec refresh function ?
4- i don't see the "Clean Alert", is it only visible in Home when the device show "C" on his screen ?

If i can provide you any log, or doing any test ...
In all cases, again thanks for your work ! 
Best

SeB

**Comment by Caldis** [comment]
@MacDyson909 yeah thanks for your suggestion, I name this plugin as `homebridge-airdog-air-purifier-x7sm` just because I  only have chance to test my plugin on it, because I own it...

for your problem, I think I can offer an option for disable the those un-supported sensors
as the PM Value, seems the device only can provide the value when the device is On, I will figure out how to auto refreshed it
for the Clean Alert, I think its a rare usage so I think its no need to support

**Comment by MacDyson909** [comment]
Hi,
Thank you for your answer !
If you can add a disable option for the sensoriel, that should be good, if not, it’s ok like that ! I can remote my airdogS ! This is most important ;-)
Again thanks for your job !

SeB

________________________________
De : Caldis_Chen <notifications@github.com>
Envoyé : Sunday, February 21, 2021 7:51:10 AM
À : Caldis/homebridge-airdog-air-purifier-x7sm <homebridge-airdog-air-purifier-x7sm@noreply.github.com>
Cc : MacDyson909 <seb4seb@gmail.com>; Mention <mention@noreply.github.com>
Objet : Re: [Caldis/homebridge-airdog-air-purifier-x7sm] Airdog X3,X5 comparibility (#1)


@MacDyson909<https://github.com/MacDyson909> yeah thanks for your suggestion, I name this plugin as homebridge-airdog-air-purifier-x7sm just because I only have chance to test my plugin on it, because I own it...

for your problem, I think I can offer an option for disable the other option to disable those sensors
as the PM Value, seems the device only can provide the value when the device is On
for the Clean Alert, I think its a rare usage so I think its no need to support
